src/models/constants/Offer.js

```diff
@@ -12,6 +12,9 @@ export default Object.freeze({
     requirements: {
         min_length: 1,
     },
+    vacancies: {
+        min: 1,
```

Comment:
It should probably be allowed to be 0. Thinking for the future if we ever update offers' vacancies, they can end up with 0 vacancies. With the current code, the DB wouldn't let that.

Reply:
Should I update only the DB or the validators as well?

Reply:
DB at least, validators if you want